Framework/PythonInterface/plugins/algorithms/RebinRagged.py

```diff
@@ -141,9 +143,9 @@ def PyExec(self):
             for i, (name, xmin, xmax, delta) in enumerate(zip(names, xmins, xmaxs, deltas)):
                 # don't  go beyond the range of the data
                 x = inputWS.readX(i)
-                if xmin < x[0] or xmin == Property.EMPTY_DBL:
+                if xmin == Property.EMPTY_DBL:
```

Comment:
So you allow xmin and xmax to be outside range of vector x?

Reply:
If you don't, you can't get same results inside  and outside of `AlignAndFocusPowder` because some of the TOF range disappears due to the various clippings.